eZ/Bundle/EzPublishRestBundle/Resources/config/services.yml

```diff
@@ -178,6 +178,7 @@ services:
             - "@ezpublish.api.service.location"
             - "@ezpublish.api.service.content"
             - "@ezpublish.api.service.trash"
+            - "@ezpublish.api.service.inner_url_alias"
```

Comment:
Please use `"@ezpublish.api.service.url_alias"` to pass top-most layer (in this case just for a convention, in other cases this would cause Signal Slot being omitted).